src/resources/Catalogs/ProductListingInterfaces.ts

```diff
@@ -0,0 +1,99 @@
+import {FieldOperatorType, FieldValueType} from '../Enums';
+
+export interface ProductListingsListOptions {
+    /**
+     * The page number.
+     */
+    page?: number;
+    /**
+     * The size of the page.
+     */
+    pageSize?: number;
```

Comment:
Use the paginated utils, see: https://github.com/coveo/platform-client/pull/454/

(Works in underway for standardization on the pagination, :bear:)